### ISSUE TITLE
How to get the camera parameters

### ISSUE THREAD

**Issue by hengshanji** [biubug6/Pytorch_Retinaface#37]
Hi, thanks for your excellent work, i run the detect.py and get the box and landmark result. However, i want to get the camera parameters ,like camera location,camera pose and focal length. Is there interface to do this ?

**Comment by biubug6** [comment]
Camera calibration?

**Comment by hengshanji** [comment]
No, i want  camera pose  result, like this
![image](https://user-images.githubusercontent.com/7146801/68353451-9bb5ac80-0144-11ea-84f7-be107527c530.png)

**Comment by biubug6** [comment]
Sorry, The implementation does not contain dense face regression. Official mxnet implementation also does not contain dense face regression which doesn't show much  details in paper.

**Comment by quocnhat** [comment]
hi @biubug6, 
Without the Dense branch, how can you train to get comparable results with the mxnet version? I read the paper and see that the most important things in there perhaps they included the dense regression loss to train the model.
